### PR TITLE
Fix some docs.

### PR DESCRIPTION
**Pull request by qingqing01**

### CODE REVIEW

**Review by heavengate — COMMENTED** [review]
安装文档里面的install document改到http://www.paddlepaddle.org.cn/documentation/docs/en/1.5/beginners_guide/install/index_en.html  ？

**Comment by qingqing01** [comment]
@heavengate http://www.paddlepaddle.org.cn/  我感觉首页的安装比  http://www.paddlepaddle.org.cn/documentation/docs/en/1.5/beginners_guide/install/index_en.html 这个更清晰。

**Review by willthefrog — APPROVED** [review]
LGTM

File: PaddleCV/PaddleDetection/docs/DATA.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ Loads `COCO` type datasets with directory structures like this:
   |   ...
   ```
 
-- Pascal VOC data source
-Loads `Pascal VOC` like datasets with directory structure like this:
+- PASCAL VOC data source
```
Comment (willthefrog) [inline PaddleCV/PaddleDetection/docs/DATA.md]: Both are okay.  According to my observations, "Pascal VOC" is used more often actually

Comment (qingqing01) [reply PaddleCV/PaddleDetection/docs/DATA.md]: I change to Pascal VOC

**Pull request merged**